api/src/main/java/org/pmiops/workbench/db/dao/DataSetServiceImpl.java

```diff
@@ -534,6 +537,28 @@ private String getQualifiedColumnName(Domain currentDomain, String columnName) {
         .collect(Collectors.toList());
   }
 
+  @Override
+  @Transactional
+  public DataSet cloneDataSetToWorkspace(
```

Comment:
fwiw @jmthibault79 tells me that we're not calling this `clone` anymore, but that it's `duplicate`.